### ISSUE TITLE
Agree on git workflow.

### ISSUE THREAD

**Issue by banksdev** [banksdev/minitwit#4]

**Comment by banksdev** [comment]
Proposed solution:
* master branch is the release branch
* development is the default branch
* feature branches, branch out from development with name: **feature/[feature-name]**
* features should (if possible) be directly connected to an issue

Release versioning:
* Major, Minor, Patch
e.g. v1.0.0

**Comment by ariansajina** [comment]
agreed, let's keep it simple. can we rename master to release? then everything is super self-explainable
